### PR TITLE
bw: add deposit/withdraw flag in currencies ccxt/ccxt#11107

### PR DESCRIPTION
**Pull request by sc0Vu**
Add deposit/withdraw flag in currencies 

ccxt/ccxt#11107

### CODE REVIEW

File: js/bw.js [hunk 1]
```diff
@@ -292,6 +292,8 @@ module.exports = class bw extends Exchange {
                 'info': currency,
                 'name': code,
                 'active': active,
+                'deposit': undefined,
+                'withdraw': undefined,
```
Comment (kroitor) [inline js/bw.js:296]: I think `drawFee`, `dailyDrawLimit` and `drawFlag` are all about withdrawing, we should most likely use the drawFlag to determine the state of `withdraw` here.

File: js/bw.js [hunk 2]
```diff
@@ -292,6 +292,8 @@ module.exports = class bw extends Exchange {
                 'info': currency,
                 'name': code,
                 'active': active,
+                'deposit': undefined,
```
Comment (kroitor) [inline js/bw.js:295]: I think we should use `rechargeFlag` to determine if `deposit` is `true` or `false`.

Comment (sc0Vu) [reply js/bw.js:295]: Should we also change active to this?
```JS
const active = deposit && withdraw &&  state === 1;
```

Comment (kroitor) [reply js/bw.js]: I would keep it as is for now.

**Pull request merged**